bin/setup.sh

```diff
@@ -23,7 +23,7 @@ parse_network () { # $1: network, $2: use_fork
             CHAIN_ID=44787
             export FOUNDRY_PROFILE=alfajores-deployment
             ;;
-        "celo")
+        "mainnet")
```

Comment:
nit: Celo is more explicit than mainnet, I personally would change this back. What was the motivation for changing it in the first place?

Reply:
I did it because deploy bash script descriptions suggested mainnet when choosing the network but it didn't exist in setUp therefore the script wouldn't run. So I'll just update the descriptions then.

Reply:
Yeah it was my bad, I changed the code but didn't update the description.